pkg/infrastructure/services/messaging_test.go

```diff
@@ -44,7 +50,14 @@ func Test_fetchRequiredTopicsWithEmptyReply(t *testing.T) {
 	emptyResponse := "[]"
 	instance := createServiceInstance(t)
 
-	server := mockKogitoSvcReplies(t, emptyResponse)
+	handlers := []serverHandler{
+		{
+			Path:         topicInfoPath,
+			JSONResponse: emptyResponse,
+		},
+	}
+
+	server := mockKogitoSvcReplies(t, handlers)
```

Comment:
Don't need a temp with a declared slice :)

```suggestion
	server := mockKogitoSvcReplies(t, serverHandler{{Path: topicInfoPath, JSONResponse: emptyResponse}})
```